.github/workflows/tag.yml

```
@@ -0,0 +1,104 @@
+# NB: release notes inspired by https://blogs.sap.com/2018/06/22/generating-release-notes-from-git-commit-messages-using-basic-shell-commands-gitgrep/
+name: Tag Creator
+on:
+  issues:
+    types: [opened]
+jobs:
+  bumpversion:
+    if: >-
+      (
+        startsWith( github.event.issue.title, '[bumpversion:major]' ) ||
+        startsWith( github.event.issue.title, '[bumpversion:minor]' ) ||
+        startsWith( github.event.issue.title, '[bumpversion:patch]' )
+      ) && (
+        github.event.issue.user.login == 'lukasheinrich' ||
+        github.event.issue.user.login == 'matthewfeickert' ||
+        github.event.issue.user.login == 'kratsg'
+      )
+    env:
+      IS_MAJOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:major]' ) }}
+      IS_MINOR: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:minor]' ) }}
+      IS_PATCH: >-
+          ${{ startsWith( github.event.issue.title, '[bumpversion:patch]' ) }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: determine part
+      run: |
+        if [ $IS_MAJOR == 'true' ]
+        then
+          echo "::set-env name=BV_PART::major"
+        elif [ $IS_MINOR == 'true' ]
+        then
+          echo "::set-env name=BV_PART::minor"
+        else
+          echo "::set-env name=BV_PART::patch"
+        fi
+    - name: add bumpversion/${{ env['BV_PART'] }} label
+      uses: actions/github@v1.0.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        args: label bumpversion/${{ env['BV_PART'] }}
+    - uses: actions/checkout@master
+    - name: Prepare repository
+      run: git checkout "${GITHUB_REF:11}"
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: install bumpversion
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install bumpversion
+    - name: set up git user
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+    - name: run bumpversion ${{ env['BV_PART'] }}
```

```suggestion
    - name: Run bumpversion ${{ env['BV_PART'] }}
```

Some of the step names are capitalized, but not all, so go all start with caps?